app/components/team-card.tsx

```diff
@@ -0,0 +1,111 @@
+import {
+	MilitaryTech,
+	TrendingUp,
+	TrendingUpOutlined
+} from "@mui/icons-material";
+import { Box, ButtonBase, Grid, Typography } from "@mui/material";
+import React from "react";
+import MaetIcon from "./maet-icon";
+
+export type TeamCardProps = {
```

Comment:
The teamCard and PlayerCard types are the same. maybe make a 'Card' type and import it to these files. If you think the types might change in the future you can leave them separate.